content/log/explore.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Explore
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+## Overview 
+
+The Log explorer is an optimized view which makes it easy to do any kind of troubleshooting and data exploration.
```

Comment:
> The Log explorer is an optimized view which makes it easy to do any kind of troubleshooting and data exploration.

Optimized how? Unless we say how, it's just a buzzword.

How about:

"The Log explorer is your home base for troubleshooting and exploration."

Reply:
👍